app/index.html

```diff
@@ -90,7 +94,7 @@
                         <a class="page-scroll" href="explore.html">Explore</a>
                     </li>
                     <li>
-                        <a class="page-scroll" href="login.html" onclick = "initalizeFirebase();firebase.auth().signOut()">Logout</a>
+                        <a class="page-scroll" href="login.html" onclick = "firebase.auth().signOut()">Logout</a>
```

Comment:
As long as I test, it functions well. I can logout and get to login.html as well as modify the profile information by making the changes.